src/Features/Core/Portable/AddFileBanner/AbstractAddFileBannerCodeRefactoringProvider.cs

```diff
@@ -78,7 +78,7 @@ public override async Task ComputeRefactoringsAsync(CodeRefactoringContext conte
             {
                 cancellationToken.ThrowIfCancellationRequested();
 
-                var siblingBanner = await TryGetBannerAsync(siblingDocument, siblingRoot, cancellationToken).ConfigureAwait(false);
+                var siblingBanner = await TryGetBannerAsync(siblingDocument, siblingRoot!, cancellationToken).ConfigureAwait(false);
```

Comment:
`TryGetBannerAsync` should be updated to have a signature of 
```csharp
Task<ImmutableArray<SyntaxTrivia>> TryGetBannerAsync(Document document, SyntaxNode? root, CancellationToken cancellationToken)
```

instead of suppressing this nullable warning. The implementation already checks if its null.